src/calendar/view/CalendarView.ts

```diff
@@ -157,7 +164,7 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 			},
 			ColumnType.Foreground,
 			{
-				minWidth: size.first_col_min_width,
+				minWidth: size.calendar_first_col_min_width,
```

Comment:
from changing settings I recall that adjusting one column did require adjusting other columns to keep breakpoints (like `desktop_layout_width` or `two_column_layout_width`) working. Do we also need to do this here or is it fine as-is?

Reply:
We have addressed this in our new commit.